core/domain/user_jobs_one_off_test.py

```diff
@@ -1482,6 +1482,12 @@ def test_for_existing_exploration_ids(self):
             migrated_subscription_model.last_updated
         )
 
+    def test_rerun(self):
+        self.test_for_empty_activity_ids()
+        self.test_for_non_empty_activity_ids()
+        self.test_for_multiple_models()
+        self.test_for_existing_exploration_ids()
```

Comment:
No, do this a normal way you would write any other test. But there would be the `self._run_one_off_job()` twice in the test run.

Reply:
Done.